doc/_static/custom_math/uptime_downtime_limits.yaml

```diff
@@ -1,16 +1,22 @@
-# Here we define various different constraints to represent the same intent: to stop a technology from operating too little or too much in a year.
-# This can be used to represent technology downtime when e.g. undergoing maintenance.
-# It can also be used to represent a simplistic inability to ramp technologies up/down too often, e.g. for nuclear power plants.
-# The exact constraint you activate depends on how resolved your technologies are.
-# For instance, the two annual capacity factor constraints can work to represent a fleet of nuclear power plants which tend to operate within an annual capacity factor range of e.g., 75-85%.
-# The `downtime_period` constraint can be used if you know a specific period in time where your technology will need to be down for maintenance and can be defined by setting a value in those timesteps, leaving all other timesteps empty (i.e., NaN).
-# The `downtime_period_decision` constraint will allow a technology with integer decision variables activated to decide on a certain number of timesteps in the year to not be operating, although it is not possible to enforce that these downtime periods are consecutive.
+# title: Up-/downtime limits
 
-# New technology-level parameters
-# capacity_factor_min
-# capacity_factor_max
-# downtime_periods (from CSV as a timeseries)
-# uptime_limit
+# description: |
```

Comment:
the pipe leave the result with the line breaks when loaded in, `>` ignores the line breaks. Do we have a preference?

Reply:
The pipe is needed to keep formatting like paragraphs intact, so I think we should always go with the pipe.